README.md

```diff
@@ -228,6 +233,7 @@ Provides solutions for precision farming, enabling farmers to monitor their crop
 
 ## Research
 
+- [Blue Sky Analytics](https://blueskyhq.in) - A Big Data and AI startup building Geospatial Data Platforms for Environmental Data.
```

Comment:
"Big Data and AI" is a little buzzword-y, I think you could leave that part off.